examples/ERP/README.txt

```diff
@@ -0,0 +1,4 @@
+Classification of ERP
```

Comment:
This file should be removed, unless you provide a quantic version of ERP classification.

Reply:
Thanks for catching this out. The file is now removed.